os/kernel/binary_manager/binary_manager_bootparam.c

```diff
@@ -141,28 +142,31 @@ int binary_manager_scan_bootparam(binmgr_bpinfo_t *bp_info)
 		/* Read bootparam data */
 		ret = read(fd, (FAR uint8_t *)bootparam, BOOTPARAM_SIZE);
 		if (ret != BOOTPARAM_SIZE) {
-			printf("[Bootparam Scanning] Failed to read bootparam, errno %d\n", errno);
+			bmdbg("Fail to read BP, errno %d\n", errno);
 			continue;
 		} else if (!is_valid_bootparam(bootparam)) {
+			bmdbg("BP%d is invalid\n", bp_idx);
```

Comment:
bp_idx is already printed at start of for

Reply:
Yes, but I added it for readability